tests/datetime_test.py

```diff
@@ -167,3 +167,31 @@ def test_datetime_tz(session):
     assert result["datetime"].tzname() is None
     assert result["datetime"].timetz() == dt.time(19, 19, 30)
     assert result["datetime"].tzinfo is None
+
+
+@pytest.mark.skip("Test case not supported on SQLAlchemy 1.3")
```

Comment:
same question here about skipping the test.

Reply:
Also fixed. Thanks again.